ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -550,11 +559,10 @@ private void processNewPut(ScanResults results, IndexValue putValue) {
    * @param originalPutValue the {@link IndexValue} of the original PUT that is getting deleted
    */
   private void processNewDelete(ScanResults results, IndexValue deleteValue, IndexValue originalPutValue) {
-    long operationTimeInMs = deleteValue.getOperationTimeInMs();
-    if (operationTimeInMs == Utils.Infinite_Time) {
-      operationTimeInMs =
-          index.getIndexSegments().floorEntry(deleteValue.getOffset()).getValue().getLastModifiedTimeMs();
-    }
+    long operationTimeInMs = deleteValue.getOperationTimeInMs() == Utils.Infinite_Time ? index.getIndexSegments()
```

Comment:
sorry. Guess you can leave it the old way. That is more readable compared to this